CHANGELOG.md

```diff
@@ -1,6 +1,52 @@
 ## 8.0.0 (Unreleased)
 ### Notes:
-- Supported Controller version: **UserConnect-8.0.xxxx**
+- Supported Controller version: **UserConnect-8.0.0-1000.2432**
```

Comment:
I should have caught this earlier, we don't do UserConnect for 8.0 anymore should just be `8.0.0-1000.2432`